mod/gc_group_layout/views/default/groups/profile/summary.php

```diff
@@ -80,18 +107,22 @@
 	                'is_trusted' => true,
                     'class' => '',
                 ));
-				echo '<b>' . elgg_echo('groups:members') . ':</b> ' . $all_members_link;
+
+                if (strstr(strtolower($_SERVER['HTTP_USER_AGENT']), 'gsa-crawler') === false)
+					echo '<b>' . elgg_echo('groups:members') . ':</b> ' . $all_members_link;
```

Comment:
i wasn't sure if there was a huge difference between strong and b, i think i had just left the code as is